tests/unit/Entity/ResearchGroupTest.php

```diff
@@ -397,27 +379,6 @@ public function testGetEmailAddress()
         );
     }
 
-    /**
-     * Make sure all nullable properties can be set to null.
-     *
-     * @return void
-     */
-    public function testSetNullable()
-    {
-        $this->researchGroup->setName(null);
-        $this->researchGroup->setFundingCycle(null);
-        $this->researchGroup->setUrl(null);
-        $this->researchGroup->setPhoneNumber(null);
-        $this->researchGroup->setDeliveryPoint(null);
-        $this->researchGroup->setCity(null);
-        $this->researchGroup->setAdministrativeArea(null);
-        $this->researchGroup->setPostalCode(null);
-        $this->researchGroup->setCountry(null);
-        $this->researchGroup->setDescription(null);
-        $this->researchGroup->setLogo(null);
-        $this->researchGroup->setEmailAddress(null);
-    }
-
```

Comment:
Why removing the tests for null?

Reply:
There is no assertion for this test and it is more like testing php whether it can take null arguments